output/README.md

```diff
@@ -0,0 +1,108 @@
+This document is intended to help understand the function of the files contained within the output directory 
+of urban-weed-genomics.
+
+<!-- Do not edit the table of contents. It is updated automatically by GitHub 
+Actions. -->
+
+<!-- toc -->
```

Comment:
The automated github action I set up only generates a TOC for the bash folder markdown. Do you want to add one here? 
- If yes, I need to edit the github action accordingly
- If not, you can take all of the TOC bit out

Reply:
Thanks! I'm just going to take it out.